From ce1663f60a50c491af971ed7a73e5b99f32b3228 Mon Sep 17 00:00:00 2001
From: Stefan Lippers-Hollmann <s.l-h@gmx.de>
Date: Thu, 15 Aug 2013 21:39:51 +0200
Subject: [PATCH 61/65] acerhk: drop superfluous debugging info

Instead of using pr_info(), this can also be queried via debugfs.

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>
---
 drivers/platform/x86/acerhk.c | 61 -------------------------------------------
 1 file changed, 61 deletions(-)

--- a/drivers/platform/x86/acerhk.c
+++ b/drivers/platform/x86/acerhk.c
@@ -205,129 +205,84 @@ static void send_kbd_cmd(unsigned char c
 
 static void enable_wlan_ec_1(void)
 {
-	if (verbose)
-		pr_info("enabling WLAN via EC variant 1\n");
-
 	send_kbd_cmd(0xe7, 0x01);
 	acerhk_wlan_state = 1;
 }
 
 static void disable_wlan_ec_1(void)
 {
-	if (verbose)
-		pr_info("disabling WLAN via EC variant 1\n");
-
 	send_kbd_cmd(0xe7, 0x00);
 	acerhk_wlan_state = 0;
 }
 
 static void enable_bluetooth_ec_1(void)
 {
-	if (verbose)
-		pr_info("enabling Bluetooth via EC variant 1\n");
-
 	send_kbd_cmd(0xe7, 0x03);
 	acerhk_bluetooth_state = 1;
 }
 
 static void disable_bluetooth_ec_1(void)
 {
-	if (verbose)
-		pr_info("disabling Bluetooth via EC variant 1\n");
-
 	send_kbd_cmd(0xe7, 0x02);
 	acerhk_bluetooth_state = 0;
 }
 
 static void enable_wlan_ec_2(void)
 {
-	if (verbose)
-		pr_info("enabling WLAN via EC variant 2\n");
-
 	send_kbd_cmd(0x45, acerhk_bluetooth_state ? 0xa2 : 0xa0);
 	acerhk_wlan_state = 1;
 }
 
 static void disable_wlan_ec_2(void)
 {
-	if (verbose)
-		pr_info("disabling WLAN via EC variant 2\n");
-
 	send_kbd_cmd(0x45, acerhk_bluetooth_state ? 0xa1 : 0xa3);
 	acerhk_wlan_state = 0;
 }
 
 static void enable_bluetooth_ec_2(void)
 {
-	if (verbose)
-		pr_info("enabling Bluetooth via EC variant 2\n");
-
 	send_kbd_cmd(0x45, acerhk_wlan_state ? 0xa2 : 0xa1);
 	acerhk_bluetooth_state = 1;
 }
 
 static void disable_bluetooth_ec_2(void)
 {
-	if (verbose)
-		pr_info("disabling Bluetooth via EC variant 2\n");
-
 	send_kbd_cmd(0x45, acerhk_wlan_state ? 0xa0 : 0xa3);
 	acerhk_bluetooth_state = 0;
 }
 
 static void enable_dritek_keyboard(void)
 {
-	if (verbose)
-		pr_info("enabling dritek keyboard extension\n");
-
 	send_kbd_cmd(0x59, 0x90);
 }
 
 static void disable_dritek_keyboard(void)
 {
-	if (verbose)
-		pr_info("disabling dritek keyboard extension\n");
-
 	send_kbd_cmd(0x59, 0x91);
 }
 
 static void enable_mail_led_ec_1(void)
 {
-	if (verbose)
-		pr_info("enabling mail led via EC variant 1\n");
-
 	send_kbd_cmd(0xe8, 0x01);
 }
 
 static void disable_mail_led_ec_1(void)
 {
-	if (verbose)
-		pr_info("disabling mail led via EC variant 1\n");
-
 	send_kbd_cmd(0xe8, 0x00);
 }
 
 static void enable_mail_led_ec_2(void)
 {
-	if (verbose)
-		pr_info("enabling mail led via EC variant 2\n");
-
 	send_kbd_cmd(0x59, 0x92);
 }
 
 static void disable_mail_led_ec_2(void)
 {
-	if (verbose)
-		pr_info("disabling mail led via EC variant 2\n");
-
 	send_kbd_cmd(0x59, 0x93);
 }
 
 static void enable_mail_led_ec_3(void)
 {
-	if (verbose)
-		pr_info("enabling mail led via EC variant 3\n");
-
 	preempt_disable();
 	if (!my_i8042_wait_write())
 		outl(0x80008894, 0xCF8);
@@ -339,9 +294,6 @@ static void enable_mail_led_ec_3(void)
 
 static void disable_mail_led_ec_3(void)
 {
-	if (verbose)
-		pr_info("disabling mail led via EC variant 3\n");
-
 	preempt_disable();
 	if (!my_i8042_wait_write())
 		outl(0x80008894, 0xCF8);
@@ -1846,17 +1798,8 @@ static int acerhk_probe(struct platform_
 
 		/* do model specific initialization */
 		model_init();
-
-		/* Without a bios routine we cannot do anything except on dritek
-		   type HW, unload on other types */
-		if (verbose && bios_routine)
-			pr_info("bios routine found at 0x%x\n", bios_routine);
-
-		pr_info("Acer Travelmate hotkey driver successfully loaded.\n");
-
 #else
 		setup_model_features();
-		pr_info("Acer Travelmate hotkey driver dummy\n");
 		enable_dritek_keyboard();
 #endif
 
@@ -1918,9 +1861,6 @@ static int acerhk_remove(struct platform
 	misc_deregister(&acerhk_misc_dev);
 	disable_dritek_keyboard();
 
-	if (verbose >= 3)
-		pr_info("unloaded\n");
-
 	return 0;
 }
 
@@ -1978,7 +1918,6 @@ static void __exit acerhk_exit(void)
 {
 	platform_device_unregister(acerhk_platform_device);
 	platform_driver_unregister(&acerhk_driver);
-	pr_info("removed.\n");
 }
 
 module_init(acerhk_init);
